README.md

```diff
@@ -1 +1,3 @@
-# git101
\ No newline at end of file
+# git101
+
+Celia is cool
```

Comment:
Not sure about this one, might be able to replace it with a slightly lower Big Oh Algorithm